[Topic Entry] Sklearn decision-trees #4931

the-ak95 opened the pull request:
created md file

<!--
👋 Hi, thanks for submitting a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

<!-- Please write a summary of the change, such as which topic(s) and file(s) that you have edited or created. Please also include relevant motivation and context: -->

### Issue Solved

<!--
Please use the following format(s) to link the issue numbers that your contribution addresses:
Closes #issue-number (where `issue-number` corresponds to the PRs' respective issue)
-->

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

<!-- Please check ALL the boxes: -->

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Codecademy/docs?pullRequest=4979) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=4979) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=4979) it.</sub>

SaviDahegaonkar reviewed (changes requested):
Hey @the-ak95 ,
This PR is a concept entry and not a term entry. I'd like for you to follow these [guidelines ](https://github.com/Codecademy/docs/edit/main/documentation/concept-entry-template.md)when writing this PR. You can also check this[ issue ](https://github.com/Codecademy/docs/issues/4931)related to this PR and write it accordingly. Could you have these changes done and then I will review this file?

Thank You.

the-ak95 commented:
Sorry about that. Is there any way to change this file or do I have to make
a new one entirely? Right now I am trying to make a new file.

On Sun, Jul 28, 2024 at 3:22 AM Savi Dahegaonkar ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hey @the-ak95 <https://github.com/the-ak95> ,
> This PR is a concept entry and not a term entry. I'd like for you to
> follow these guidelines
> <https://github.com/Codecademy/docs/edit/main/documentation/concept-entry-template.md>when
> writing this PR. You can also check this issue
> <https://github.com/Codecademy/docs/issues/4931>related to this PR and
> write it accordingly.
>
> Thank You.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Codecademy/docs/pull/4979#pullrequestreview-2203452919>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRBDWMNY4SMKJEMASUKRY3ZOTA7NAVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEMBTGQ2TEOJRHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

the-ak95 commented:
I updated the file and made a new PR request. To my knowledge, the file
should be correctly updated now. Please let me know if I have missed
anything.

Thank You,
Akshat Verma

On Sun, Jul 28, 2024 at 11:17 PM Akshat Verma ***@***.***>
wrote:

> Sorry about that. Is there any way to change this file or do I have to
> make a new one entirely? Right now I am trying to make a new file.
>
> On Sun, Jul 28, 2024 at 3:22 AM Savi Dahegaonkar ***@***.***>
> wrote:
>
>> ***@***.**** requested changes on this pull request.
>>
>> Hey @the-ak95 <https://github.com/the-ak95> ,
>> This PR is a concept entry and not a term entry. I'd like for you to
>> follow these guidelines
>> <https://github.com/Codecademy/docs/edit/main/documentation/concept-entry-template.md>when
>> writing this PR. You can also check this issue
>> <https://github.com/Codecademy/docs/issues/4931>related to this PR and
>> write it accordingly.
>>
>> Thank You.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Codecademy/docs/pull/4979#pullrequestreview-2203452919>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BFRBDWMNY4SMKJEMASUKRY3ZOTA7NAVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEMBTGQ2TEOJRHE>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

SaviDahegaonkar commented:
Hey @the-ak95,
The changes done by you are not reflected at my side. Please do have look at it and let me know.

Regards,
Savi

the-ak95 commented:
Hello, I have once again updated this file and I have followed the
standards. I deeply apologize for the misunderstanding, this is my first
collaboration and I was a bit confused. Please let me know if there is
anything else I need to do.

Thank You,
Akshat Verma

On Tue, Jul 30, 2024 at 8:46 AM Savi Dahegaonkar ***@***.***>
wrote:

> Hey @the-ak95 <https://github.com/the-ak95>,
> The changes done by you are not reflected at my side. Please do have look
> at it and let me know.
>
> Regards,
> Savi
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Codecademy/docs/pull/4979#issuecomment-2258666494>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRBDWO4XLGABLNZQZQMYA3ZO6YNXAVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDENJYGY3DMNBZGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

SaviDahegaonkar commented:
> Hello, I have once again updated this file and I have followed the standards. I deeply apologize for the misunderstanding, this is my first collaboration and I was a bit confused. Please let me know if there is anything else I need to do. Thank You, Akshat Verma
> […](#)
> On Tue, Jul 30, 2024 at 8:46 AM Savi Dahegaonkar ***@***.***> wrote: Hey @the-ak95 <https://github.com/the-ak95>, The changes done by you are not reflected at my side. Please do have look at it and let me know. Regards, Savi — Reply to this email directly, view it on GitHub <[#4979 (comment)](https://github.com/Codecademy/docs/pull/4979#issuecomment-2258666494)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BFRBDWO4XLGABLNZQZQMYA3ZO6YNXAVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDENJYGY3DMNBZGQ> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Okay now worries, I will review the file and let you know if any.

SaviDahegaonkar reviewed (changes requested):
Hey @the-ak95,
 I had a look at your file and suggested you some changes. Please look for it and get it done. You can also refer to this [link](https://github.com/Codecademy/docs/tree/main/documentation) while writing this PR.
Also I would like you to have a look at the issue linked to this PR - 
The entry should include:
An introduction to the concept
A Syntax section that provides the syntax for the concept (if applicable)
An Example section that provides an example demonstrating the concept in use (if applicable)
A Codebyte Example section that provides a codebyte example demonstrating the concept in use (if applicable) all these points must be included
Thanks,
Savi

the-ak95 commented:
I have reviewed and applied these suggestions. Please let me know if there
is anything else I need to do. Thank you for your help.



On Thu, Aug 1, 2024 at 11:11 AM Savi Dahegaonkar ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hey @the-ak95 <https://github.com/the-ak95>,
> I had a look at your file and suggested you some changes. Please look for
> it and get it done. You can also refer to this link
> <https://github.com/Codecademy/docs/tree/main/documentation> while
> writing this PR.
> Also I would like you to have a look at the issue linked to this PR -
> The entry should include:
> An introduction to the concept
> A Syntax section that provides the syntax for the concept (if applicable)
> An Example section that provides an example demonstrating the concept in
> use (if applicable)
> A Codebyte Example section that provides a codebyte example demonstrating
> the concept in use (if applicable) all these points must be included
> Thanks,
> Savi
> ------------------------------
>
> In sklearndecisiontrees.md
> <https://github.com/Codecademy/docs/pull/4979#discussion_r1700560894>:
>
> > @@ -0,0 +1,42 @@
> +---
> +Title: 'Python:Sklearn Decision Trees'
>
> ⬇️ Suggested change
>
> -Title: 'Python:Sklearn Decision Trees'
> +Title: 'Python:sklearn-decision-trees'
>
> The file name and title name must be same, but the title name is always
> lowercase and we use dashes instead of spaces.
> ------------------------------
>
> In sklearndecisiontrees.md
> <https://github.com/Codecademy/docs/pull/4979#discussion_r1700570362>:
>
> > +CatalogContent:
> +  - 'getting-started-with-python-for-data-science'
> +  - 'paths/data-science'
> +  - 'machine learning'
>
> ⬇️ Suggested change
>
> -CatalogContent:
> -  - 'getting-started-with-python-for-data-science'
> -  - 'paths/data-science'
> -  - 'machine learning'
> +CatalogContent:
> +  - 'getting-started-with-python-for-data-science'
> +  - 'paths/data-science'
> +  - 'machine learning'
>
> Use the course/path that is already mentioned in the catalog content. md
> file if the required paths are not already present in it. You can add it
> there. Please have a look at this list
> <http://documentation/catalog-content.md>.
> ------------------------------
>
> In sklearndecisiontrees.md
> <https://github.com/Codecademy/docs/pull/4979#discussion_r1700586852>:
>
> > +Subjects:
> +- 'Computer Science'
> +- 'Machine Learning'
> +Tags:
> +- 'Algorithms'
> +- 'Machine Learning'
> +
> +
> +CatalogContent:
> +  - 'getting-started-with-python-for-data-science'
> +  - 'paths/data-science'
> +  - 'machine learning'
> +
> +---
> +
> +**Sklearn**, or **Scikit-learn**, is one of the few, [open-source](https://www.codecademy.com/resources/docs/open-source) machine learning library's for the [Python](https://www.codecademy.com/resources/docs/python) language. One way that Sklearn demonstrates machine learning is by using a Decision tree. A decision tree is a tool that is used to make possible decisions based on certain conditions in the world of machine learning. It is a form of a hierarchical model that has a tree-like shape. This model can chart decisions, consequences, event outcomes, resource costs, and utilities. A decision tree has several key components. The root node at the top represents the entire dataset and initiates the data split. Decision nodes represent features used for further splitting, which have multiple branches that can determine possible outcomes. Leaf nodes or terminal nodes are the endpoints that represent final outcomes. These components form a hierarchical structure which can further determine guiding decisions based on data attributes.
>
> ⬇️ Suggested change
>
> -**Sklearn**, or **Scikit-learn**, is one of the few, [open-source](https://www.codecademy.com/resources/docs/open-source) machine learning library's for the [Python](https://www.codecademy.com/resources/docs/python) language. One way that Sklearn demonstrates machine learning is by using a Decision tree. A decision tree is a tool that is used to make possible decisions based on certain conditions in the world of machine learning. It is a form of a hierarchical model that has a tree-like shape. This model can chart decisions, consequences, event outcomes, resource costs, and utilities. A decision tree has several key components. The root node at the top represents the entire dataset and initiates the data split. Decision nodes represent features used for further splitting, which have multiple branches that can determine possible outcomes. Leaf nodes or terminal nodes are the endpoints that represent final outcomes. These components form a hierarchical structure which can further determine guiding decisions based on data attributes.
> +**Sklearn**, or **Scikit-learn**, is one of the few[open-source](https://www.codecademy.com/resources/docs/open-source) machine learning libraries for the [Python](https://www.codecademy.com/resources/docs/python) language. One way that Sklearn demonstrates machine learning is by using a decision tree. A decision tree is a tool used to make decisions based on certain conditions in the world of machine learning. It is a hierarchical model that has a tree-like shape. This model can chart decisions, consequences, event outcomes, resource costs, and utilities. A decision tree has several key components. The root node at the top represents the entire dataset and initiates the data split. Decision nodes represent features used for further splitting and have multiple branches that determine possible outcomes. Leaf nodes or terminal nodes are the endpoints that represent outcomes. These components form a hierarchical structure that can further determine guiding decisions based on data attributes.
>
> There are some minor grammatical errors that I have corrected and you
> should keep this in mind while writing this kind of entry. I think you can
> also add an image in your entry to properly understand how a decision tree
> looks and how it works, it can make the concept easier to understand.
> ------------------------------
>
> In sklearndecisiontrees.md
> <https://github.com/Codecademy/docs/pull/4979#discussion_r1700592739>:
>
> > +- 'Algorithms'
> +- 'Machine Learning'
> +
> +
> +CatalogContent:
> +  - 'getting-started-with-python-for-data-science'
> +  - 'paths/data-science'
> +  - 'machine learning'
> +
> +---
> +
> +**Sklearn**, or **Scikit-learn**, is one of the few, [open-source](https://www.codecademy.com/resources/docs/open-source) machine learning library's for the [Python](https://www.codecademy.com/resources/docs/python) language. One way that Sklearn demonstrates machine learning is by using a Decision tree. A decision tree is a tool that is used to make possible decisions based on certain conditions in the world of machine learning. It is a form of a hierarchical model that has a tree-like shape. This model can chart decisions, consequences, event outcomes, resource costs, and utilities. A decision tree has several key components. The root node at the top represents the entire dataset and initiates the data split. Decision nodes represent features used for further splitting, which have multiple branches that can determine possible outcomes. Leaf nodes or terminal nodes are the endpoints that represent final outcomes. These components form a hierarchical structure which can further determine guiding decisions based on data attributes.
> +
> +## Computer Science
> +
> +Sklearn demonstrates machine learning by using computer science languages such as Python. Python is a type of computer science language that is very popular in the computer science world.  Machine learning is also a branch of computer science and it ideally falls under the same category.
>
> ⬇️ Suggested change
>
> -Sklearn demonstrates machine learning by using computer science languages such as Python. Python is a type of computer science language that is very popular in the computer science world.  Machine learning is also a branch of computer science and it ideally falls under the same category.
> +Sklearn demonstrates machine learning by using programming languages such as Python. Python is a type of language that is very popular in the computer science world.  Machine learning is a branch of computer science and it ideally falls under the same category.
>
> Use of "computer science language" This word doesn't seem more accurate
> here you can instead use "programming languages" for better readability.
> ------------------------------
>
> In sklearndecisiontrees.md
> <https://github.com/Codecademy/docs/pull/4979#discussion_r1700594163>:
>
> > +
> +
> +
> +
>
> Do not use more extra spaces while writing an entry. You can have a space
> of one or two lines but avoid using this much longer space.
> ------------------------------
>
> In sklearndecisiontrees.md
> <https://github.com/Codecademy/docs/pull/4979#discussion_r1700596497>:
>
> > +CatalogContent:
> +  - 'getting-started-with-python-for-data-science'
> +  - 'paths/data-science'
> +  - 'machine learning'
> +
> +---
> +
> +**Sklearn**, or **Scikit-learn**, is one of the few, [open-source](https://www.codecademy.com/resources/docs/open-source) machine learning library's for the [Python](https://www.codecademy.com/resources/docs/python) language. One way that Sklearn demonstrates machine learning is by using a Decision tree. A decision tree is a tool that is used to make possible decisions based on certain conditions in the world of machine learning. It is a form of a hierarchical model that has a tree-like shape. This model can chart decisions, consequences, event outcomes, resource costs, and utilities. A decision tree has several key components. The root node at the top represents the entire dataset and initiates the data split. Decision nodes represent features used for further splitting, which have multiple branches that can determine possible outcomes. Leaf nodes or terminal nodes are the endpoints that represent final outcomes. These components form a hierarchical structure which can further determine guiding decisions based on data attributes.
> +
> +## Computer Science
> +
> +Sklearn demonstrates machine learning by using computer science languages such as Python. Python is a type of computer science language that is very popular in the computer science world.  Machine learning is also a branch of computer science and it ideally falls under the same category.
> +
> +## Machine Learning
> +
> +Sklearn is a open learn library that contains tools on machine learning. For example, a type of tool is a decision tree. Decision trees are  models that can chart decisions, consequences, event outcomes, resource costs, and utilities.
>
> ⬇️ Suggested change
>
> -Sklearn is a open learn library that contains tools on machine learning. For example, a type of tool is a decision tree. Decision trees are  models that can chart decisions, consequences, event outcomes, resource costs, and utilities.
> +Sklearn is an open-source library that contains tools for machine learning. For example, one type of tool is a decision tree. Decision trees are models that can chart decisions, consequences, event outcomes, resource costs, and utilities.
>
> You had a few grammatical errors here. I have corrected it, but please
> make a note of it.
> ------------------------------
>
> In sklearndecisiontrees.md
> <https://github.com/Codecademy/docs/pull/4979#discussion_r1700600818>:
>
> > +You can find information about Sklearn's decision trees at https://scikit-learn.org/stable/modules/tree.html:
> +
> +## Sklearn Installation
> +
> +To install Sklearn, use [`pip`](https://www.codecademy.com/resources/docs/python/pip):
> +
> +```shell
> +pip install scikit-learn
>
> ⬇️ Suggested change
>
> -You can find information about Sklearn's decision trees at https://scikit-learn.org/stable/modules/tree.html:
> -
> -## Sklearn Installation
> -
> -To install Sklearn, use [`pip`](https://www.codecademy.com/resources/docs/python/pip):
> -
> -```shell
> -pip install scikit-learn
> +You can find information about Sklearn's decision trees at https://scikit-learn.org/stable/modules/tree.html.
> +
> +## Sklearn Installation
> +
> +To install Sklearn, use [`pip`](https://www.codecademy.com/resources/docs/python/pip):
> +
> +```shell
> +pip install scikit-learn
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Codecademy/docs/pull/4979#pullrequestreview-2213449824>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRBDWJ4WNBHN7E22WEKR6TZPJ25PAVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEMJTGQ2DSOBSGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

SaviDahegaonkar reviewed (approved):
Hey Author,
I hereby approve this PR for a second review as I can't find any new changes that should be made.


Thanks

mamtawardhani commented:
Hey @the-ak95, your file location for this entry is incorrect, can you please transfer the file to the following path: `docs/content/sklearn/concepts/decision-trees/decision-trees.md`; so that I can start the review?
Thank you 😄

mamtawardhani commented:
@the-ak95 are you still working on this? Please make sure that if you don't revert back within 5 days, I'll have to close this PR and this won't get merged.

the-ak95 commented:
Yes, I am working on it. I will get back to you as soon as possible.

On Fri, Aug 16, 2024 at 2:13 AM Mamta Wardhani ***@***.***>
wrote:

> @the-ak95 <https://github.com/the-ak95> are you still working on this?
> Please make sure that if you don't revert back within 5 days, I'll have to
> close this PR and this won't get merged.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Codecademy/docs/pull/4979#issuecomment-2293141007>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRBDWPLBAMQVYK4UJQDQI3ZRW7B7AVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJTGE2DCMBQG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

mamtawardhani commented:
> Yes, I am working on it. I will get back to you as soon as possible.
> […](#)
> On Fri, Aug 16, 2024 at 2:13 AM Mamta Wardhani ***@***.***> wrote: @the-ak95 <https://github.com/the-ak95> are you still working on this? Please make sure that if you don't revert back within 5 days, I'll have to close this PR and this won't get merged. — Reply to this email directly, view it on GitHub <[#4979 (comment)](https://github.com/Codecademy/docs/pull/4979#issuecomment-2293141007)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BFRBDWPLBAMQVYK4UJQDQI3ZRW7B7AVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJTGE2DCMBQG4> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Hey @the-ak95 its been 4 days, could you let me know until when can you change the path? We need to close this asap. Thank you.

the-ak95 commented:
I will change it by the end of the week. Maybe earlier. Thank you.

On Tue, Aug 20, 2024 at 4:41 AM Mamta Wardhani ***@***.***>
wrote:

> Yes, I am working on it. I will get back to you as soon as possible.
> … <#m_-8587192802317874037_>
> On Fri, Aug 16, 2024 at 2:13 AM Mamta Wardhani *@*.*> wrote: @the-ak95
> <https://github.com/the-ak95> https://github.com/the-ak95
> <https://github.com/the-ak95> are you still working on this? Please make
> sure that if you don't revert back within 5 days, I'll have to close this
> PR and this won't get merged. — Reply to this email directly, view it on
> GitHub <#4979 (comment)
> <https://github.com/Codecademy/docs/pull/4979#issuecomment-2293141007>>, or
> unsubscribe
> https://github.com/notifications/unsubscribe-auth/BFRBDWPLBAMQVYK4UJQDQI3ZRW7B7AVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJTGE2DCMBQG4
> <https://github.com/notifications/unsubscribe-auth/BFRBDWPLBAMQVYK4UJQDQI3ZRW7B7AVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJTGE2DCMBQG4>
> . You are receiving this because you were mentioned.Message ID: @.*>
>
> Hey @the-ak95 <https://github.com/the-ak95> its been 4 days, could you
> let me know until when can you change the path? We need to close this asap.
> Thank you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Codecademy/docs/pull/4979#issuecomment-2298650087>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRBDWIOYFT5BJVYHXVRIMTZSMTONAVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJYGY2TAMBYG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

mamtawardhani commented:
> I will change it by the end of the week. Maybe earlier. Thank you. On Tue, Aug 20, 2024 at 4:41 AM Mamta Wardhani ***@***.***> wrote:
> […](#)
> Yes, I am working on it. I will get back to you as soon as possible. … <#m_-8587192802317874037_> On Fri, Aug 16, 2024 at 2:13 AM Mamta Wardhani *@*.*> wrote: @the-ak95 <https://github.com/the-ak95> https://github.com/the-ak95 <https://github.com/the-ak95> are you still working on this? Please make sure that if you don't revert back within 5 days, I'll have to close this PR and this won't get merged. — Reply to this email directly, view it on GitHub <#4979 (comment) <[#4979 (comment)](https://github.com/Codecademy/docs/pull/4979#issuecomment-2293141007)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/BFRBDWPLBAMQVYK4UJQDQI3ZRW7B7AVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJTGE2DCMBQG4 <https://github.com/notifications/unsubscribe-auth/BFRBDWPLBAMQVYK4UJQDQI3ZRW7B7AVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJTGE2DCMBQG4> . You are receiving this because you were mentioned.Message ID: @.*> Hey @the-ak95 <https://github.com/the-ak95> its been 4 days, could you let me know until when can you change the path? We need to close this asap. Thank you. — Reply to this email directly, view it on GitHub <[#4979 (comment)](https://github.com/Codecademy/docs/pull/4979#issuecomment-2298650087)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BFRBDWIOYFT5BJVYHXVRIMTZSMTONAVCNFSM6AAAAABLSF3QC2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJYGY2TAMBYG4> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Hey @the-ak95 any update? I need to close this by tuesday max

mamtawardhani commented:
Closing this since the author has created another PR.
